package.json

```
@@ -5,12 +5,14 @@
   "scripts": {
     "serve": "vuepress dev",
     "build": "vuepress build",
-    "lint": "markdownlint . --ignore node_modules --ignore index.md"
+    "lint": "markdownlint . --ignore node_modules --ignore index.md",
+    "sc": "mdspell -a -n -x -r --en-us"
```

I think canada generally uses british spelling? https://www.ryerson.ca/content/dam/studentlearningsupport/resources/grammar-handouts/Canadian_English.pdf (though I've seen both, but I think `en-us` would be less correct than `en-gb`)

```suggestion
    "spellcheck": "mdspell -a -n -x -r --en-gb"
```

---

Looks like the gb dictionary has more words than the us one too.

---

they call it _great_ britain for a reason bud